README.md

```diff
@@ -27,3 +27,13 @@ is temporary. The final name will be selected later.
   * For running:
     - Java Runtime Environment (JRE) version 11 or later.
     - PROJ 6.
+
+## Contributing
+
+Please submit issues or pull requests on the Github [project
+page](https://github.com/Kortforsyningen/PROJ-JNI].
```

Comment:
On the [home page](https://github.com/willcohen/PROJ-JNI/tree/contributing-coc) the link appears in clear. Maybe because there is a line break between "project" and "page"?